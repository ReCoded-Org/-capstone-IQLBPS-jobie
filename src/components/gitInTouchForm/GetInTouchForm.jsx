import React from 'react';

export default function () {
  return (
    <div>
      {' '}
      <div className="flex justify-around     ">
        <div className=" flex flex-col justify-evenly">
          <div className="div-1 m-8">
            <h2 className="text-4xl text-slate-600 subpixel-antialiased font-semibold">
              Get in touch
            </h2>
          </div>
          <div className="flex justify-around">
            <div className="flex flex-col">
              <div className="flex justify-evenly gap-3">
                <label
                  htmlFor="Firstname"
                  className="text-left text-xl font-semibold"
                >
                  {' '}
                  first name <br />
                  <input type="text" className="border-2 border-solid p-1.5" />
                </label>

                <label
                  htmlFor="Lastname"
                  className="text-left text-xl font-semibold"
                >
                  {' '}
                  Last name <br />{' '}
                  <input type="text" className="border-2 border-solid p-1.5" />
                </label>
              </div>
              <div className=" flex-col text-left">
                <label
                  htmlFor="Lastname"
                  className="text-left text-xl font-semibold"
                >
                  {' '}
                  Email <br />{' '}
                  <input type="text" className="border-2 border-solid p-1.5" />
                </label>
                <br />{' '}
                <label
                  htmlFor="Lastname"
                  className="text-left text-xl font-semibold"
                >
                  <br /> Messege <br />{' '}
                  <textarea
                    name=""
                    className="border-2 border-solid"
                    id=""
                    cols="30"
                    rows="3"
                  >
                    {' '}
                  </textarea>
                </label>
                <br />{' '}
                <button
                  type="button"
                  className="bg-accent  text-white font-bold py-3 px-7 rounded-full"
                >
                  Send
                </button>
              </div>
            </div>
          </div>
        </div>
        <div>
          <div className="div-2 mt-10 ">
            <h1 className="text-4xl subpixel-antialiased text-slate-600 font-semibold mb-10">
              Contact Us
            </h1>
            <br />
            <div className=" flex-col text-2xl text-center   gap-10">
              <div className="flex flex-row justify-center  gap-3 mb-7">
                {' '}
                <svg
                  xmlns="http://www.w3.org/2000/svg"
                  fill="none"
                  viewBox="0 0 24 24"
                  strokeWidth={1.5}
                  stroke="currentColor"
                  className="w-6 h-6"
                >
                  <path
                    strokeLinecap="round"
                    strokeLinejoin="round"
                    d="M15 10.5a3 3 0 11-6 0 3 3 0 016 0z"
                  />
                  <path
                    strokeLinecap="round"
                    strokeLinejoin="round"
                    d="M19.5 10.5c0 7.142-7.5 11.25-7.5 11.25S4.5 17.642 4.5 10.5a7.5 7.5 0 1115 0z"
                  />
                </svg>
                <p>As Sulaimaniyah ,Iraq </p>{' '}
              </div>
              <div className="flex flex-row justify-center gap-3 mb-7">
                <svg
                  xmlns="http://www.w3.org/2000/svg"
                  fill="none"
                  viewBox="0 0 24 24"
                  strokeWidth={1.5}
                  stroke="currentColor"
                  className="w-6 h-6"
                >
                  <path
                    strokeLinecap="round"
                    strokeLinejoin="round"
                    d="M2.25 6.75c0 8.284 6.716 15 15 15h2.25a2.25 2.25 0 002.25-2.25v-1.372c0-.516-.351-.966-.852-1.091l-4.423-1.106c-.44-.11-.902.055-1.173.417l-.97 1.293c-.282.376-.769.542-1.21.38a12.035 12.035 0 01-7.143-7.143c-.162-.441.004-.928.38-1.21l1.293-.97c.363-.271.527-.734.417-1.173L6.963 3.102a1.125 1.125 0 00-1.091-.852H4.5A2.25 2.25 0 002.25 4.5v2.25z"
                  />
                </svg>
                <p> +964 750 113 0495 </p>{' '}
              </div>
              <div className="flex flex-row justify-center gap-3 mb-7">
                {' '}
                <svg
                  xmlns="http://www.w3.org/2000/svg"
                  fill="none"
                  viewBox="0 0 24 24"
                  strokeWidth={1.5}
                  stroke="currentColor"
                  className="w-6 h-6"
                >
                  <path
                    strokeLinecap="round"
                    d="M16.5 12a4.5 4.5 0 11-9 0 4.5 4.5 0 019 0zm0 0c0 1.657 1.007 3 2.25 3S21 13.657 21 12a9 9 0 10-2.636 6.364M16.5 12V8.25"
                  />
                </svg>
                <p> jobie.contact@.com </p>{' '}
              </div>
              <div className="flex flex-row justify-center gap-3 mb-7 ">
                <svg
                  xmlns="http://www.w3.org/2000/svg"
                  fill="none"
                  viewBox="0 0 24 24"
                  strokeWidth={1.5}
                  stroke="currentColor"
                  className="w-6 h-6"
                >
                  <path
                    strokeLinecap="round"
                    strokeLinejoin="round"
                    d="M12 6v6h4.5m4.5 0a9 9 0 11-18 0 9 9 0 0118 0z"
                  />
                </svg>
                <p>9:00 - 17:00</p>{' '}
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
}
