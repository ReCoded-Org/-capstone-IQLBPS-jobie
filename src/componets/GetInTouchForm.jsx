import React from 'react'

export default function ()   {  return (
    <div>    <div className="flex  justify-around max-w-full mt-[60px] ">
                <div className="div-1 max-w-[50%]"><h2 className='text-2xl subpixel-antialiased'>Get in touch</h2></div>
                <div className="div-2 max-w-[50%]"> <h2 className='text-2xl'>Contact Us</h2></div>
    </div>
        <div className='flex justify-around'>
            <div className='flex flex-col'>
            <div className='flex justify-around gap-3' >
                <label htmlFor="Firstname" className='text-left text-xl font-semibold'> first name <br/><input type="text" className='border-2 border-solid p-1.5' /></label>
                
             <label htmlFor="Lastname"className='text-left text-xl font-semibold'> first name <br/> <input type="text" className='border-2 border-solid p-1.5'/></label>
                </div>
                <div className=" flex-col text-left">
             <label htmlFor="Lastname"className='text-left text-xl font-semibold'> Email <br/> <input type="text" className='border-2 border-solid p-1.5'/></label>  
 <br/> <label htmlFor="Lastname"className='text-left text-xl font-semibold'> first name <br/> <textarea name="" className='border-2 border-solid' id="" cols="30" rows="3"> </textarea></label>  
    </div>
           </div>
        <div className='flex justify-center'><ul><li>one</li>
        <li>two</li>
        <li>there</li>
                <li>Four</li></ul></div></div>
         
    </div>
  )} 

 
