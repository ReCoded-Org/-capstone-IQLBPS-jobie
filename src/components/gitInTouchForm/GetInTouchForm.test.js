import React from 'react';
import renderer from 'react-test-renderer';

import GetInTouchForm from './GetInTouchForm';

it('renders Get in touch form corectlly  correctly', () => {
  const tree = renderer
    .create(<GetInTouchForm/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
