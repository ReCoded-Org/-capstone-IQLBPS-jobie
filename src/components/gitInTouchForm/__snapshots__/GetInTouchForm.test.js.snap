// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Get in touch form corectlly  correctly 1`] = `
<div
  className="flex  justify-center my-[100px] gap-3 small:flex-col "
>
  <div
    className="flex-1 justify-center   m-5 small:m-5"
  >
    <h1
      className="text-4xl subpixel-antialiased text-slate-600 font-semibold"
    >
       
      Get in touch
    </h1>
    <form
      className="flex flex-col gap-3 mt-10"
      onSubmit={[Function]}
    >
      <div
        className="flex flex-col  gap-3"
      >
        <div
          className="flex flex-auto justify-center  gap-2 small:flex-col medium:flex-col"
        >
          <label
            className="flex flex-col"
            htmlFor="name"
          >
             
            <p
              className="text-left text-lg subpixel-antialiased text-slate-600 font-semibold"
            >
              Name
               
            </p>
            <input
              className="px-6 py-2   border-2 border-gray-900"
              id="n44ame"
              name="firstName"
              onBlur={[Function]}
              onChange={[Function]}
              placeholder="Name"
              type="text"
            />
            <p
              className="text-sm text-red-500 text-left"
            />
          </label>
          <label
            className="flex flex-col"
            htmlFor="name"
          >
             
            <p
              className="text-left text-lg subpixel-antialiased text-slate-600 font-semibold"
            >
               
              Last name
               
            </p>
            <input
              className="px-6 py-2   border-2 border-gray-900"
              id="nam3e"
              name="lastName"
              onBlur={[Function]}
              onChange={[Function]}
              placeholder="Last Name"
              type="text"
            />
            <p
              className="text-sm text-red-500 text-left"
            />
          </label>
           
        </div>
      </div>
      <div
        className="mx-[125px] small:mx-0 medium:mx-0"
      >
         
        <label
          className="flex justify-center flex-col "
          htmlFor="email"
        >
          <p
            className="text-left text-lg subpixel-antialiased text-slate-600 font-semibold"
          >
            Email :
          </p>
          <input
            className="px-3 py-2   border-2 border-gray-900"
            id="name1"
            name="email"
            onBlur={[Function]}
            onChange={[Function]}
            placeholder="example@email.com"
            type="text"
          />
          <p
            className="text-sm text-red-500 text-left"
          />
        </label>
      </div>
      <div
        className="mx-[125px] small:mx-0 medium:mx-0"
      >
         
        <label
          className="flex justify-center flex-col "
          htmlFor="email"
        >
          <p
            className="text-left text-lg subpixel-antialiased text-slate-600 font-semibold"
          >
            Message :
          </p>
          <textarea
            className="border-2 border-gray-900"
            cols="30"
            name="message"
            onBlur={[Function]}
            onChange={[Function]}
            rows="3"
          />
          <p
            className="text-sm text-red-500 text-left"
          />
        </label>
      </div>
      <div
        className="flex justify-start"
      >
        <button
          className="bg-accent ml-[150px] mt-5  text-white font-bold hover:bg-red-500 py-3 px-8 rounded-full small:ml-0 medium:ml-0"
          disabled={false}
          type="submit"
        >
          Send
        </button>
      </div>
    </form>
  </div>
  <div
    className="flex-1   m-5 "
  >
    <h1
      className="text-4xl subpixel-antialiased text-slate-600 font-semibold mb-10"
    >
      Contact Us
    </h1>
    <div
      className=" flex-col text-2xl text-center   gap-10"
    >
      <div
        className="flex flex-row justify-center  gap-3 mb-7"
      >
         
        <svg
          className="w-6 h-6"
          fill="none"
          stroke="currentColor"
          strokeWidth={1.5}
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M15 10.5a3 3 0 11-6 0 3 3 0 016 0z"
            strokeLinecap="round"
            strokeLinejoin="round"
          />
          <path
            d="M19.5 10.5c0 7.142-7.5 11.25-7.5 11.25S4.5 17.642 4.5 10.5a7.5 7.5 0 1115 0z"
            strokeLinecap="round"
            strokeLinejoin="round"
          />
        </svg>
        <p>
          As Sulaimaniyah ,Iraq 
        </p>
         
      </div>
      <div
        className="flex flex-row justify-center gap-3 mb-7"
      >
        <svg
          className="w-6 h-6"
          fill="none"
          stroke="currentColor"
          strokeWidth={1.5}
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M2.25 6.75c0 8.284 6.716 15 15 15h2.25a2.25 2.25 0 002.25-2.25v-1.372c0-.516-.351-.966-.852-1.091l-4.423-1.106c-.44-.11-.902.055-1.173.417l-.97 1.293c-.282.376-.769.542-1.21.38a12.035 12.035 0 01-7.143-7.143c-.162-.441.004-.928.38-1.21l1.293-.97c.363-.271.527-.734.417-1.173L6.963 3.102a1.125 1.125 0 00-1.091-.852H4.5A2.25 2.25 0 002.25 4.5v2.25z"
            strokeLinecap="round"
            strokeLinejoin="round"
          />
        </svg>
        <p>
           +964 750 113 0495 
        </p>
         
      </div>
      <div
        className="flex flex-row justify-center gap-3 mb-7"
      >
         
        <svg
          className="w-6 h-6"
          fill="none"
          stroke="currentColor"
          strokeWidth={1.5}
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M16.5 12a4.5 4.5 0 11-9 0 4.5 4.5 0 019 0zm0 0c0 1.657 1.007 3 2.25 3S21 13.657 21 12a9 9 0 10-2.636 6.364M16.5 12V8.25"
            strokeLinecap="round"
          />
        </svg>
        <p>
           jobie.contact@.com 
        </p>
         
      </div>
      <div
        className="flex flex-row justify-center gap-3 mb-7 "
      >
        <svg
          className="w-6 h-6"
          fill="none"
          stroke="currentColor"
          strokeWidth={1.5}
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M12 6v6h4.5m4.5 0a9 9 0 11-18 0 9 9 0 0118 0z"
            strokeLinecap="round"
            strokeLinejoin="round"
          />
        </svg>
        <p>
          9:00 - 17:00
        </p>
         
      </div>
    </div>
  </div>
</div>
`;
